src/css/main.css

```diff
@@ -56,10 +55,14 @@ html, body {
     align-items: center;
     width: 100%;
     padding: 0.5em 0em 0.5em 0em;
-    background-color: #dddd00;
+    background-color: rgb(255, 209, 102);
     color: black;
 }
 
+.aside-hour-location-container-center-padding{
+    padding: 0em 20em 0em 20em;
```

Comment:
Don't do this. It's just an element to create padding so this should be a class/id for the element that needs padding itself not an independent element to create space. Also on a side note, your padding could be simpler: `padding: 0em 20em;`. This way it will apply the same values for the omitted sides (bottom and left side in this case).